contents/install/update/major/0.4-to-0.5.md

```diff
@@ -0,0 +1,46 @@
+1. Run `bundle exec sidekiq`, wait 5 minutes and take it down again with `Ctrl+C`.
+1. If you modified `public/default.html`, `public/404.html`, `public/422.html` or `public/500.html` take a backup of the changes, they will be gone or overwritten. Run `git checkout Gemfile.lock db/schema.rb public/*.html`. Run `git stash` to preserve any other uncommitted changes. You can later restore stashed changes with `git stash pop`.
+1. Update RVM with `rvm get stable`. Install Ruby 2.1 with `rvm install 2.1`.
+1. Update diaspora with `git pull; git checkout v0.5.10.2`.
+1. Activate the new Ruby with `cd .. && cd -`.
+1. Install the latest Bundler with `gem install bundler --version 1.12.6`.
+1. Install the dependencies with `bin/bundle install --full-index --without development test`. CAREFUL: Do not run any commands that are suggested at the end of the terminal output; that was already done for you.
+1. Verify you didn't run any additional commands already done for you, then run `git status` and delete any untracked files residing in `db/migrate`.
+1. Edit `config/initializers/secret_token.rb`, replace `secret_token` with `secret_key_base`:
+
+   ~~~ruby
+   # Old
+   Rails.application.config.secret_token = '***********...'
+
+   # New
+   Diaspora::Application.config.secret_key_base = '*************...'
+   ~~~
+1. If you're using MySQL/MariaDB, edit `config/database.yml`: Replace `charset: utf8` with `encoding: utf8mb4`, notice how the key changed! Replace `collation: utf8_bin` with `collation: utf8mb4_bin`. Have a look at the updated `config/database.yml.example` if you need additional guidance in this step.
+1. Run `bin/rake db:migrate`, this will take quite a while but you should occasionally check if it failed and seek assistance if it happened to fail.
+1. Run `bin/rake tmp:cache:clear assets:precompile`. This steps now generates `public/404.html`, `public/422.html` and `public/500.html`.
+1. You can now start diaspora* again as you're used to.
+
+You should now compare your `config/diaspora.yml` to the updated `config/diaspora.yml.example`, for example with `vimdiff`. Notable changes, not only limited to the configuration, are:
+
+* The surrounding markup for the custom splash page changed slightly, please refer to the [changelog][changelog].
+* If you're using Paypal donations, the settings in `diaspora.yml` for it changed, in order to support the unhosted button. Please take a look at the updated `diaspora.yml.example`.
+* The default for including jQuery from a CDN has changed. If you want to continue to include it from a CDN, please explicitly set the `jquery_cdn` setting to true in `diaspora.yml`.
+* There's a new feature to automatically prune inactive accounts, if you want to enable it please refer to the [changelog][changelog].
+* There's a new feature to proxy embedded images from external sources through your server, strengthening your users privacy. If you want to enable it please refer to Camo.
+
+TODO: Link Camo
```

Comment:
👀

Reply:
Known. Camo will be part of the Podmin Guides, which I have not yet done. However, my editor has a global `TODO` view, so I won't forget this. :)